schematics/ng-add/index.ts

```diff
@@ -29,12 +32,6 @@ import {
   getSingleSpaAngularDependency,
   getAngularBuildersCustomWebpackDependency,
 } from './dependencies';
-import {
-  findPropertyInAstObject,
-  insertPropertyInAstObjectInOrder,
-  removePropertyInAstObject,
-} from '@schematics/angular/utility/json-utils';
```

Comment:
This hasn't been a public API and was removed in Angular 11.

Reply:
Sorry @arturovt I've been the guy introducing these imports... For next time, how can I know if it is part of the public API or not ?